br/pkg/rtree/rtree_test.go

```diff
@@ -85,92 +78,93 @@ func (s *testRangeTreeSuite) TestRangeTree(c *C) {
 			{StartKey: []byte("d"), EndKey: []byte("")},
 		})
 
-	c.Assert(search([]byte{}), IsNil)
-	c.Assert(search([]byte("a")), DeepEquals, rangeA)
-	c.Assert(search([]byte("b")), IsNil)
-	c.Assert(search([]byte("c")), DeepEquals, rangeC)
-	c.Assert(search([]byte("d")), IsNil)
+	require.Nil(t, search([]byte{}))
+	require.Equal(t, rangeA, search([]byte("a")))
+	require.Nil(t, search([]byte("b")))
+	require.Equal(t, rangeC, search([]byte("c")))
+	require.Nil(t, search([]byte("d")))
 
 	rangeTree.Update(*rangeB)
-	c.Assert(rangeTree.Len(), Equals, 3)
-	c.Assert(search([]byte("b")), DeepEquals, rangeB)
+	require.Equal(t, 3, rangeTree.Len())
+	require.Equal(t, rangeB, search([]byte("b")))
 	assertIncomplete([]byte(""), []byte(""),
 		[]rtree.Range{
 			{StartKey: []byte(""), EndKey: []byte("a")},
 			{StartKey: []byte("d"), EndKey: []byte("")},
 		})
 
 	rangeTree.Update(*rangeD)
-	c.Assert(rangeTree.Len(), Equals, 4)
-	c.Assert(search([]byte("d")), DeepEquals, rangeD)
+	require.Equal(t, 4, rangeTree.Len())
+	require.Equal(t, rangeD, search([]byte("d")))
 	assertIncomplete([]byte(""), []byte(""), []rtree.Range{
 		{StartKey: []byte(""), EndKey: []byte("a")},
 	})
 
 	// None incomplete for any range after insert range 0
 	rangeTree.Update(*range0)
-	c.Assert(rangeTree.Len(), Equals, 5)
+	require.Equal(t, 5, rangeTree.Len())
 
 	// Overwrite range B and C.
 	rangeBD := newRange([]byte("b"), []byte("d"))
 	rangeTree.Update(*rangeBD)
-	c.Assert(rangeTree.Len(), Equals, 4)
+	require.Equal(t, 4, rangeTree.Len())
 	assertAllComplete()
 
 	// Overwrite range BD, c-d should be empty
 	rangeTree.Update(*rangeB)
-	c.Assert(rangeTree.Len(), Equals, 4)
+	require.Equal(t, 4, rangeTree.Len())
 	assertIncomplete([]byte(""), []byte(""), []rtree.Range{
 		{StartKey: []byte("c"), EndKey: []byte("d")},
 	})
 
 	rangeTree.Update(*rangeC)
-	c.Assert(rangeTree.Len(), Equals, 5)
+	require.Equal(t, 5, rangeTree.Len())
 	assertAllComplete()
 }
 
-func (s *testRangeTreeSuite) TestRangeIntersect(c *C) {
+func TestRangeIntersect(t *testing.T) {
+	t.Parallel()
 	rg := newRange([]byte("a"), []byte("c"))
 
 	start, end, isIntersect := rg.Intersect([]byte(""), []byte(""))
-	c.Assert(isIntersect, Equals, true)
-	c.Assert(start, DeepEquals, []byte("a"))
-	c.Assert(end, DeepEquals, []byte("c"))
+	require.Equal(t, true, isIntersect)
+	require.Equal(t, []byte("a"), start)
+	require.Equal(t, []byte("c"), end)
 
 	start, end, isIntersect = rg.Intersect([]byte(""), []byte("a"))
-	c.Assert(isIntersect, Equals, false)
-	c.Assert(start, DeepEquals, []byte(nil))
-	c.Assert(end, DeepEquals, []byte(nil))
+	require.Equal(t, false, isIntersect)
+	require.Equal(t, []byte(nil), start)
+	require.Equal(t, []byte(nil), end)
 
 	start, end, isIntersect = rg.Intersect([]byte(""), []byte("b"))
-	c.Assert(isIntersect, Equals, true)
-	c.Assert(start, DeepEquals, []byte("a"))
-	c.Assert(end, DeepEquals, []byte("b"))
+	require.Equal(t, true, isIntersect)
```

Comment:
```suggestion
	require.True(t, isIntersect)
```

and so on for the remaining 5.